js/src/components/free-listings/configure-product-listings/shipping-time/shipping-time-setup/index.scss

```diff
@@ -5,7 +5,7 @@
 		margin-bottom: calc(var(--main-gap) / 2);
 
 		.countries-time-input-form {
-			max-width: 300px;
+			max-width: $gla-size-medium;
```

Comment:
💅 
`$gla-size-small` and `$gla-size-medium` are general naming but most dimension usages are not general layouts. Perhaps we could use a smaller and more specific name.

Reply:
Probably I would use `$gla-width-shipping-form`.

Reply:
Problem with that is that if we want to use the size for another form, a modal or any other size definition it won't fit. 

So I would suggest to go for a more generic and flexible naming.